ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/ExternalBackupIntegrations/icons.tsx

```diff
@@ -0,0 +1,229 @@
+                        integration instead. See the AWS documentation about S3 endpoints for a
+                        complete list of AWS endpoints.
+                    </a>
+                </div>
+            }
+            headerContent={'AWS S3 endpoint'}
+        >
+            <button
+                type="button"
+                aria-label="More info for input"
+                onClick={(e) => e.preventDefault()}
+                aria-describedby="simple-form-name-01"
+                className="pf-v5-c-form__group-label-help"
+            >
+                <HelpIcon />
+            </button>
+        </Popover>
+    );
+}
+
+export function s3RegionIcon(): ReactElement {
+    return (
+        <Popover
+            bodyContent={
+                <div>
+                    <a
+                        href="https://docs.aws.amazon.com/general/latest/gr/s3.html"
+                        target="_blank"
+                        rel="noreferrer"
+                    >
+                        See the AWS documentation about S3 endpoints for a complete list of AWS
+                        regions.
+                    </a>
+                </div>
+            }
+            headerContent={'AWS S3 region'}
+        >
+            <button
+                type="button"
+                aria-label="More info for input"
+                onClick={(e) => e.preventDefault()}
+                aria-describedby="simple-form-name-01"
+                className="pf-v5-c-form__group-label-help"
+            >
+                <HelpIcon />
+            </button>
+        </Popover>
+    );
+}
+
+export function s3IamRole(): ReactElement {
+    return (
+        <Popover
+            bodyContent={
+                <div>
+                    <a
+                        href="https://docs.openshift.com/acs/integration/integrate-using-short-lived-tokens.html"
+                        target="_blank"
+                        rel="noreferrer"
+                    >
+                        Enables authentication via short-lived tokens using AWS Secure Token
+                        Service. See the Red Hat RHACS documentation for more information.
+                    </a>
+                </div>
+            }
+            headerContent={'AWS container IAM role'}
+        >
+            <button
+                type="button"
+                aria-label="More info for input"
+                onClick={(e) => e.preventDefault()}
+                aria-describedby="simple-form-name-01"
+                className="pf-v5-c-form__group-label-help"
+            >
+                <HelpIcon />
+            </button>
+        </Popover>
+    );
+}
+
+export function s3CompatibleEndpointIcon(): ReactElement {
+    return (
+        <Popover
+            bodyContent={
+                <div>
+                    Modifies the endpoint under which the S3 compatible service is reached. Must be
+                    reachable via https.
```

Comment:
Should we add a text here to redirect to the `Amazon S3` in case someone is trying to use the `S3 API Compatible` with an AWS S3? Something like what you have in the `s3EndpointIcon`: 
```
Note that when using an AWS S3, it is recommended to create an *Amazon S3* integration instead.
```